app/assets/javascripts/analytics/ecommerce.js

```diff
@@ -0,0 +1,66 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Ecommerce = function (config) {
+    this.init = function (element) {
+      this.forEachEcommerceRow(element, this._sendImpression);
+      this.forEachEcommerceRow(element, this._trackProductClick);
+    }
+
+    this.forEachEcommerceRow = function (element, fct) {
+      var ecommerceRowCss = '[data-ecommerce-row]';
+      var ecommerceRows = element.find(ecommerceRowCss);
+      var startPosition = parseInt(element.data('ecommerce-start-index'));
+
+      ecommerceRows.each(function(index, ecommerceRow) {
+        var $ecommerceRow = $(ecommerceRow);
+
+        var content_id = $ecommerceRow.attr('data-ecommerce-content-id'),
```

Comment:
Not sure why this isn't caught by a linter, but shouldn't `content_id` be `contentId` throughout?